batchUploadResourceCollections.sh

```diff
@@ -1,20 +1,31 @@
 #!/bin/bash
 
-# This bash script is used to batch upload resource collections to the iHub server. It iterates through 
+# This bash script is used to batch upload resource collections to the iHub server. It iterates through  the directories in the src/main/flowResources directory and uploads each resource collection to the server.
 
 # Assumes that you are in the root of the community directory and that it's named like /ihub-community-<COMMUNITY_NAME>
 community=$(basename "$(pwd)" | grep -o '[^-]*$') 
 
 cd ./src/main/flowResources || exit
 
+errorReport=""
 for dir in ./*; do
     if [ -d "$dir" ]; then
         resource=$(basename $dir)
         echo ""
         echo "Uploading collection $resource"
-        uploadResourceCollection.sh "$resource" "$community"
+        response=$(uploadResourceCollection.sh "$resource" "$community")
```

Comment:
I'm almost positive that `response` will be the output of the `uploadResourceCollection .sh` command.  If you're gaol is to obtain the exit code from the `uploadResourceCollection.sh`, `uploadResourceCollection.sh` does not set the exit code correctly (it always appears to be a successful 0).